### PR TITLE
Conflict PR not triggering test runs info added

### PR DESCRIPTION
**Pull request by ma-zah**

### CODE REVIEW

File: test-results.mdx [hunk 1]
```diff
@@ -29,14 +29,21 @@ Once tests are executed, you will receive a comment in your GitHub / Azure DevOp
...
-| status ⌛️   | execution in progress                                     |
-| status ❌   | execution failed                                          |
-| status 💔   | something went wrong on our side during execution         |
-| details     | link which takes you to the test case details (see below) |
+| column      | description           |
+| ----------- | --------------------- |
+| description | name of the test case |
+| status ✅   | execution successful  |
+
+// eslint-disable-next-line prettier/prettier
```
Comment (Germandrummer92) [inline test-results.mdx:37]: ???

Comment (Germandrummer92) [reply test-results.mdx:37]: why?

**Review by Germandrummer92 — CHANGES_REQUESTED** [review]

**Comment by Germandrummer92** [comment]
also this somehow includes the changes from the other PR you will get some conflicts like that ;)

File: test-results.mdx (hunk 1 above)
Comment (ma-zah) [reply test-results.mdx:37]: i don't remember exactly, but with this change, the linter didn't protest. Linter forces me to do things.

**Pull request closed without merging**